clients/src/main/java/org/apache/kafka/clients/consumer/KafkaShareConsumer.java

```diff
@@ -497,7 +497,7 @@ public void acknowledge(ConsumerRecord<K, V> record) {
      * <p>This method can only be used if the consumer is using <b>explicit acknowledgement</b>.
      *
      * @param record The record to acknowledge
-     * @param type The acknowledgement type which indicates whether it was processed successfully
+     * @param type The acknowledge type which indicates whether it was processed successfully
```

Comment:
if `acknowledge type` is a term used in shared consumer, could you please consider fixing it in the codebase?

https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareFetch.java#L113

https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareFetch.java#L132